configuration/sitemaps.md

```diff
@@ -581,29 +581,29 @@ Note also, the effect of omitting `Temperature` and the comparison operator in t
-| fuchsia    | *`► #ff00ff`*{: style="color: #ff00ff"} |
-| white      | *`► #ffffff`*{: style="color: #ffffff"} |
-| lime       | *`► #00ff00`*{: style="color: #00ff00"} |
-| green      | *`► #008000`*{: style="color: #008000"} |
-| navy       | *`► #000080`*{: style="color: #000080"} |
-| blue       | *`► #0000ff`*{: style="color: #0000ff"} |
-| teal       | *`► #008080`*{: style="color: #008080"} |
-| aqua       | *`► #00ffff`*{: style="color: #00ffff"} |
-| black      | *`► #000000`*{: style="color: #000000"} |
-| silver     | *`► #c0c0c0`*{: style="color: #c0c0c0"} |
-| gray       | *`► #808080`*{: style="color: #808080"} |
+| Color Name | Preview                                       | RGB Color Code |
+|------------|-----------------------------------------------|----------------|
+| maroon     | <div style="color: #800000;">&#11044;</div>   | `#800000`      |
+| red        | <div style="color: #ff0000;">&#11044;</div>   | `#ff0000`      |
+| orange     | <div style="color: #ffa500;">&#11044;</div>   | `#ffa500`      |
+| olive      | <div style="color: #808000;">&#11044;</div>   | `#808000`      |
+| yellow     | <div style="color: #ffff00;">&#11044;</div>   | `#ffff00`      |
+| purple     | <div style="color: #800080;">&#11044;</div>   | `#800080`      |
+| fuchsia    | <div style="color: #ff00ff;">&#11044;</div>   | `#ff00ff`      |
+| white      | <div style="color: #ffffff;">&#11044;</div>   | `#ffffff`      |
+| lime       | <div style="color: #00ff00;">&#11044;</div>   | `#00ff00`      |
+| green      | <div style="color: #008000;">&#11044;</div>   | `#008000`      |
+| navy       | <div style="color: #000080;">&#11044;</div>   | `#000080`      |
+| blue       | <div style="color: #0000ff;">&#11044;</div>   | `#0000ff`      |
+| teal       | <div style="color: #008080;">&#11044;</div>   | `#008080`      |
+| aqua       | <div style="color: #00ffff;">&#11044;</div>   | `#00ffff`      |
+| black      | <div style="color: #000000;">&#11044;</div>   | `#000000`      |
+| silver     | <div style="color: #c0c0c0;">&#11044;</div>   | `#c0c0c0`      |
+| gray       | <div style="color: #808080;">&#11044;</div>   | `#808080`      |
 
 Please take note that colors other than those listed in the list above may be used.
-Generally, you can expected that valid HTML colors will be accepted (e.g. "green", "lightgrey", "#334455"), but note that a UI may only accept internally defined colors, or work with a special theme.
-The color names above are agreed on between all openHAB UIs and are therefor your safest choice.
+Generally, you can expected that valid HTML colors will be accepted (e.g. `green`, `lightgrey`, `#334455`), but note that a UI may only accept internally defined colors, or work with a special theme.
+The color names above are agreed on between all openHAB UIs and are therefore your safest choice. Colors defined by a human-readable name may be adjusted for higher contrast, e.g. on a dark theme `black` may be displayed as white.
```

Comment:
One sentence per line.
Also: "black may be white" is a bit surprising. Can you think of a better way to explain this?

Reply:
Better?